claim/README.md

```diff
@@ -83,21 +97,38 @@ can trust the endpoint if necessary.
 Claiming can be performed through a SOCKS5 proxy. To do this when calling the script directly supply the proxy
 endpoint as:
 
-```
+```bash
 netdata-claim.sh -token=MYTOKEN1234567 -rooms=room1,room2 -proxy=socks5h://127.0.0.1:11081
 ```
 
 When claiming via the `netdata` binary set the following options in the config:
```

Comment:
Why does claiming via the binary require a socks5 proxy but the script does not - this doesn't make sense.

Reply:
Can you explain your concern a bit more? I'm not sure I follow. The command shows explicitly setting a proxy via the `-proxy=` argument.

Reply:
The text has changed a bit and I can't see the problem. I think it was the context of the sentence but it seems to work ok now.

Reply:
Cool. Thanks for taking another peek.